app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -295,7 +295,7 @@ case class KafkaManagedOffsetCache(clusterContext: ClusterContext
               lastUpdateTimeMillis = System.currentTimeMillis()
             } catch {
               case e: Exception =>
-                warn("Failed to process a message from offset topic!", e)
+                warn(s"Failed to process a message from offset topic on cluster ${clusterContext.config.name}!", e)
```

Comment:
Do not want to hide it, but if you access a variable that doesn't exist in the Exception case then what will catch that exception?